src/Database/Redis/Sentinel.hs

```diff
@@ -46,7 +46,6 @@ import           Control.Exception     (Exception, IOException, evaluate, throwI
 import           Control.Monad
 import           Control.Monad.Catch   (Handler (..), MonadCatch, catches, throwM)
 import           Control.Monad.Except
-import           Control.Monad.IO.Class(MonadIO(liftIO))
```

Comment:
This import is needed on mtl-2.3, because re-exports are removed. If this line was removed to avoid a warning, I think it would be better to just make all imports use explicit import lists, instead of open imports.

That way, the code will be easier to read, and compatible with both mtl-2.2 and mtl-2.3, and there will be no warning.

Reply:
There was a trick with applicative and prelude after re-exports were added that
```
import Control.Applicative
import Preludee
```
worked without warnings. Maybe we can write:
```
import Control.Monad.IO.Class(MonadIO(liftIO))
import Control.Monad.Except -- the import order changed in order to avoid
```
It may not work well with code formatters, but it should fix a warning and does not require CPP.